contracts/Strategy.sol

```diff
@@ -236,6 +236,12 @@ contract Strategy is BaseStrategy {
     {
         for (uint256 i = 0; i < kashiPairs.length; i++) {
             KashiPairInfo memory kashiPairInfo = kashiPairs[i];
+            if (
+                kashiFractionTotal(
+                    kashiPairInfo.kashiPair,
+                    kashiPairInfo.pid
+                ) <= dustThreshold
+            ) continue; // skip if there's only dust
```

Comment:
minor edge case:

if the strategy assumes to be using accrued kashiPairs, can this be a problem for those KashiPairs that hold dust if they are interacted with in any way?

Reply:
Potentially, but liquidateKashiPair also calls accrue, so it would only lead to underestimating assets and leaving dust ...